README.md

```diff
@@ -13,11 +13,133 @@ Featherity is a fork of [Feather Icons](https://github.com/feathericons/feather)
 * [Contributing](#contributing)
 * [License](#license)
 
+## Installation
+
+### Package Managers
+
+``` bash
+npm install featherity
+#or
+yarn add featherity
+```
+
+### CDN
+
+``` html
+<!-- Development version -->
+<script src="https://unpkg.com/featherity@latest/dist/umd/featherity.js"></script>
+
+<!-- Production version -->
+<script src="https://unpkg.com/featherity@latest"></script>
+```
+
 ## Usage
 
 At its core, Featherity is a collection of [SVG](https://svgontheweb.com/#svg) files. This means that you can use Feather icons in all the same ways you can use SVGs (e.g. `img`, `background-image`, `inline`, `object`, `embed`, `iframe`). Here's a helpful article detailing the many ways SVGs can be used on the web: [SVG on the Web – Implementation Options](https://svgontheweb.com/#implementation)
 
-The following are additional ways you can use Featherity.
+The following are additional ways you can use Featherity.<br>
```

Comment:
Use two spaces at the end of the line instead of a `<br>`

Reply:
Thanks!